committee/sig-governance/SIG-GOVERNANCE.md

```diff
@@ -1,75 +1,80 @@
 # SIG Governance
 
-The full name of SIG is Special Interest Group. Please refer to [Community Organization](/GOVERNANCE.md#developer-group-structure). SIGs mainly aggregate a group of Active Contributors, and conducts in-depth research & contributions to one or more TiKV modules, and is promoted to Reviewer, Committer within the SIG.
+This charter details on the governance guidelines for TiKV SIGs. It will be updated as needed to meet the current needs of the TiKV project.
 
-## SIG Role and Organizational Governance
+## Organizational Governance
 
-The SIG follows the role definitions in the [GOVERNANCE](/GOVERNANCE.md). The SIG is subject to the following rules:
+In order to standardize Special Interest Group efforts, create maximum transparency, and route contributors to the appropriate SIG, SIGs should follow these guidelines:
 
-- Before starting a SIG, the charter will be established in advance. Please refer [Create the SIG charter](#Create-the-SIG-charter) to create the charter 
-- Except for holidays, the regular meeting is organized at least once every two weeks.
+- Create a charter and have it approved according to the SIG charter  process. Please refer [Create the SIG charter](#create-the-sig-charter) for more details.
+- Meet regularly, at least for 30 minutes every 2 weeks, except for long holidays
+- Keep up-to-date meeting notes, linked from the SIG's page in the community repo
+- Report activity in the weekly community meeting at least once every quarter
```

Comment:
How to report?

Reply:
This should be specified in the specific SIG charter